imports/server/loadBalance.ts

```diff
@@ -39,14 +39,16 @@ const getWorkersCount = () => {
   return parseInt(maybeWorkerCountString, 10) || 0;
 };
 
+// eslint-disable-next-line import/prefer-default-export
+export const workersCount = getWorkersCount();
+
 WebApp.onListening(() => {
-  const workersCount = getWorkersCount();
   if (workersCount > 0) {
     http.globalAgent.maxSockets = 99999;
     const proxy = HttpProxy.createProxyServer({
       xfwd: true,
     });
-    const workers = WorkerPool.create(workersCount);
+    workers = WorkerPool.create(workersCount);
```

Comment:
not sure why this is no longer `const workers`?

Reply:
Oh whoops, wilfix